packages/site-kit/components/Hero.svelte

```diff
@@ -12,7 +12,7 @@
 	<div class="hero-container">
 		<div class="hero-text">
 			<img alt="{title} logotype" width="400" height="50" class="logotype" src={logotype} />
```

Comment:
should the `alt` here just be `{title}` instead of `{title} logotype`?

Reply:
Updated in latest commit.